### PR TITLE
JWWEB-441 Minor visual issues on Payment Password screen

### PR DESCRIPTION
**Pull request by Dragonesis**

### CODE REVIEW

File: src/components/base/JInput/JInput.js [hunk 1]
```diff
@@ -140,7 +140,7 @@ class JInput extends PureComponent<Props> {
           <div className='lock'>
             <JIcon
               size='medium'
-              color='gray'
+              color={color}
```
Comment (IViol) [inline src/components/base/JInput/JInput.js]: there is shouldn't be lock icon on blue screens

File: src/components/PasswordField/Indicator/passwordFieldIndicator.scss [hunk 2]
```diff
@@ -9,9 +9,11 @@ $password-field-indicator-height: 6px;
...
   right: $form-field-white-horizontal-padding;
   width: 60px;
-  top:
-    $form-field-padding-top +
-    (($form-field-white-font-size - $password-field-indicator-height) / 2);
+  top: 30px;
+
+  &.-offset-left {
+    right: $icon-medium-size / 1.5 + $form-field-white-horizontal-padding;
+  }
```
Comment (IViol) [inline src/components/PasswordField/Indicator/passwordFieldIndicator.scss]: -offset-left but you change right attribute, looks weird

File: src/components/PasswordField/Indicator/passwordFieldIndicator.scss [hunk 3]
```diff
@@ -9,9 +9,11 @@ $password-field-indicator-height: 6px;
   position: absolute;
   right: $form-field-white-horizontal-padding;
   width: 60px;
-  top:
-    $form-field-padding-top +
-    (($form-field-white-font-size - $password-field-indicator-height) / 2);
+  top: 30px;
```
Comment (IViol) [inline src/components/PasswordField/Indicator/passwordFieldIndicator.scss]: we have password fields on blue and gray screens (inputs with different height), so `top` couldn't be fixed value

Comment (Dragonesis) [reply src/components/PasswordField/Indicator/passwordFieldIndicator.scss]: They are the same and equal to 65px

File: src/components/base/JInput/JInput.js (hunk 1 above)
Comment (stcherenkov) [reply src/components/base/JInput/JInput.js]: In my opinion, all except this is optional

**Pull request merged**